.gitignore

```diff
@@ -179,3 +179,134 @@ $RECYCLE.BIN/
+# NUNIT
+*.VisualState.xml
+# Build Results of an ATL Project
+[Dd]ebugPS/
+[Rr]eleasePS/
+dlldata.c
+# DNX
+project.lock.json
+project.fragment.lock.json
+artifacts/
+**/Properties/launchSettings.json
+*_i.h
+*.svclog
+# Chutzpah Test files
+_Chutzpah*
+*.opendb
+*.VC.db
+*.VC.VC.opendb
+*.sap
+# TFS 2012 Local Workspace
+$tf/
+*.DotSettings.user
+# JustCode is a .NET coding add-in
+.JustCode
+# Visual Studio code coverage results
+*.coverage
+*.coveragexml
+_NCrunch_*
+nCrunchTemp_*
+# MightyMoose
+*.mm.*
+AutoTest.Net/
+# Web workbench (sass)
+.sass-cache/
+*.azurePubxml
+# TODO: Comment the next line if you want to checkin your web deploy settings
+# but database connection strings (with potential passwords) will be unencrypted
+*.pubxml
+*.publishproj
+# Microsoft Azure Web App publish settings. Comment the next line if you want to
+# checkin your Azure Web App publish settings, but sensitive information contained
+# in these scripts will be unencrypted
+PublishScripts/
+# NuGet Packages
+*.nupkg
+# The packages folder can be ignored because of Package Restore
+**/packages/*
+# except build/, which is used as an MSBuild target.
+!**/packages/build/
+# Uncomment if necessary however generally it will be regenerated when needed
+#!**/packages/repositories.config
+# NuGet v3's project.json files produces more ignoreable files
+*.nuget.props
+*.nuget.targets
+# Microsoft Azure Build Output
+csx/
+# Microsoft Azure Emulator
+ecf/
+rcf/
+# Windows Store app package directories and files
+BundleArtifacts/
+Package.StoreAssociation.xml
+_pkginfo.txt
+# Visual Studio cache files
+# files ending in .cache can be ignored
+*.[Cc]ache
+# but keep track of directories ending in .cache
+!*.[Cc]ache/
+*.dbproj.schemaview
+*.jfm
+node_modules/
+orleans.codegen.cs
+# Since there are multiple workflows, uncomment next line to ignore bower_components
+# (https://github.com/github/gitignore/pull/1529#issuecomment-104372622)
+#bower_components/
+# Backup & report files from converting an old project file
+# to a newer Visual Studio version. Backup files are not needed,
+# because we have git ;-)
+# Business Intelligence projects
+*.rdl.data
+*.bim.layout
+*.bim_*.settings
+# Microsoft Fakes
+FakesAssemblies/
+# GhostDoc plugin setting file
+*.GhostDoc.xml
+# Node.js Tools for Visual Studio
+.ntvs_analysis.dat
+# Visual Studio 6 build log
+*.plg
+# Visual Studio 6 workspace options file
+*.opt
+# Visual Studio 6 auto-generated workspace file (contains which files were open etc.)
+*.vbw
+# Visual Studio LightSwitch build output
+**/*.HTMLClient/GeneratedArtifacts
+**/*.DesktopClient/GeneratedArtifacts
+**/*.DesktopClient/ModelManifest.xml
+**/*.Server/GeneratedArtifacts
+**/*.Server/ModelManifest.xml
+_Pvt_Extensions
+# Paket dependency manager
+.paket/paket.exe
+paket-files/
+# FAKE - F# Make
+.fake/
+# JetBrains Rider
+.idea/
+*.sln.iml
+# CodeRush
+.cr/
+# Python Tools for Visual Studio (PTVS)
+__pycache__/
+*.pyc
+# Cake - Uncomment if you are using it
+tools/
```

Comment:
This works because the latest cake bootstraps will download a packages.config if one is not found. However, we have been following the practice of controlling the version of Cake used through the packages.config. Without that, I think we would be giving up our control of what Cake version we are using in each project.

Also, I have found that nunit console needs to be specified here rather than in a #tools comment in order for the NUnit3 command to work on Linux.

The upshot is that I think we need to add !tools/packages.config